packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -451,7 +451,7 @@ export const getLinksSupplier =
           internalLink: link.internal,
           scopedVars: variables,
           field,
-          range: {} as any,
+          range: link.internal.range ?? ({} as any),
```

Comment:
Is it the same range as in the panel? If yes, could we pass the range from `PanelQueryRunner`? e.g.:

```diff
 series: applyFieldOverrides({
+    timeRange: data.timeRange,
     timeZone: data.request?.timezone ?? 'browser',
     data: processedData.series,
     ...fieldConfig!,
   }),
```

And later pass the range to `getLinksSupplier`.

This way it could work for any data link, without a need to pass the range in the config. Not sure if that would work for your use case.

Reply:
Yeah, it is the same time range although the plugin uses the `PanelRenderer` which calls `useFieldOverrides` and doesn't run `PanelQueryRunner`, so I don't think that'd work for our situation. Maybe scenes will fix this out of the box.